samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/DrumPlayer.kt

```diff
@@ -64,30 +65,14 @@ class DrumPlayer {
     // asset-based samples
     fun loadWavAssets(assetMgr: AssetManager): Boolean {
         var allAssetsCorrect = true
-        if (!loadWavAsset(assetMgr, "KickDrum.wav", BASSDRUM, PAN_BASSDRUM)) {
-            allAssetsCorrect = false
-        }
-        if (!loadWavAsset(assetMgr, "SnareDrum.wav", SNAREDRUM, PAN_SNAREDRUM)) {
-            allAssetsCorrect = false
-        }
-        if (!loadWavAsset(assetMgr, "CrashCymbal.wav", CRASHCYMBAL, PAN_CRASHCYMBAL)) {
-            allAssetsCorrect  = false
-        }
-        if (!loadWavAsset(assetMgr, "RideCymbal.wav", RIDECYMBAL, PAN_RIDECYMBAL)) {
-            allAssetsCorrect  = false
-        }
-        if (!loadWavAsset(assetMgr, "MidTom.wav", MIDTOM, PAN_MIDTOM)) {
-            allAssetsCorrect  = false
-        }
-        if (!loadWavAsset(assetMgr, "LowTom.wav", LOWTOM, PAN_LOWTOM)) {
-            allAssetsCorrect  = false
-        }
-        if (!loadWavAsset(assetMgr, "HiHat_Open.wav", HIHATOPEN, PAN_HIHATOPEN)) {
-            allAssetsCorrect  = false
-        }
-        if (!loadWavAsset(assetMgr, "HiHat_Closed.wav", HIHATCLOSED, PAN_HIHATCLOSED)) {
-            allAssetsCorrect  = false
-        }
+        allAssetsCorrect = loadWavAsset(assetMgr, "KickDrum.wav", BASSDRUM, PAN_BASSDRUM) && allAssetsCorrect
```

Comment:
Nice.

Reply:
Ack. Don't know how that slipped in there.